app/src/main/res/layout/administrator_controls_download_permissions_view.xml

```diff
@@ -103,6 +103,7 @@
       android:background="@color/component_color_admin_controls_options_background_color"
       android:clickable="true"
       android:focusable="true"
+      android:visibility="@{viewModel.isVisible ? View.VISIBLE: View.GONE}"
```

Comment:
I think the only thing is missing is tests to verify that this correctly shows/doesn't show when the parameter is enabled/disabled, repsectively.

Reply:
This feedback has been addressed.